packages/client-sdk-web/src/cache-client.ts

```diff
@@ -19,7 +21,8 @@ export class CacheClient extends AbstractCacheClient implements ICacheClient {
   constructor(props: CacheClientProps) {
     const controlClient: IControlClient = createControlClient(props);
     const dataClient: IDataClient = createDataClient(props);
-    super(controlClient, [dataClient]);
+    const pingClient: IPingClient = createPingClient(props);
```

Comment:
can we do this lazily?  i.e., the class can have a nullable local variable for it, and then you can create an accessor function for it that checks to see if it's undefined and instantiates the client once if it is undefined.  That way we aren't opening an extra socket connection for every instantiated CacheClient when 90% of the time people aren't going to use the ping API?